cms/src/main/java/uk/nhs/digital/arc/json/ManifestWrapper.java

```diff
@@ -0,0 +1,49 @@
+package uk.nhs.digital.arc.json;
+
+import com.fasterxml.jackson.annotation.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ManifestWrapper {
+
+    @JsonProperty("document_base")
+    private String documentBase;
+    @JsonProperty("pages")
+    private List<Page> pages = null;
+    @JsonIgnore
+    private Map<String, Object> additionalProperties = new HashMap<String, Object>();
```

Comment:
You can reduce the verbosity of this map to protected Map<String, Object> additionalProperties = new HashMap<>(); as the compiler will infer the type